src/applications/representatives/constants/sis.js

```diff
@@ -0,0 +1,8 @@
+export {
+  CLIENT_IDS,
+  OAUTH_KEYS as QUERY_PARAM_KEYS,
+} from '~/platform/utilities/oauth/constants';
+
+export {
+  logoutUrlSiS as logoutUrl,
+} from '~/platform/utilities/oauth/utilities';
```

Comment:
The reason for this name is because this module collects properties to expose an entity that represents `SIS`, so for example it'd be the difference between `person.firstName` and `person.personFirstName`. In this case, how would `SIS.logoutUrlSiS` provide clarity and avoid confusion? The module could just export one object, but I think that semantically one can provide multiple exports from a module and consider the module itself to represent some entity.